packages/i18n-calypso/src/i18n.js

```diff
@@ -191,16 +204,13 @@ I18N.prototype.geolocateCurrencySymbol = async function ( callback ) {
 	callback?.( 'string' === typeof geoData?.country_short ? geoData.country_short : '' );
 };
 
-I18N.prototype.on = function ( ...args ) {
-	this.stateObserver.on( ...args );
-};
-
-I18N.prototype.off = function ( ...args ) {
-	this.stateObserver.off( ...args );
+I18N.prototype.subscribe = function ( callback ) {
+	this.subscribers.push( callback );
+	return () => removeOne( this.subscribers, callback );
 };
 
-I18N.prototype.emit = function ( ...args ) {
-	this.stateObserver.emit( ...args );
+I18N.prototype.emitChange = function () {
+	this.subscribers.forEach( ( callback ) => callback() );
```

Comment:
`emitChange` doesn't really need to be a public method. Can we avoid the publicity somehow?

Reply:
> `emitChange` doesn't really need to be a public method. Can we avoid the publicity somehow?

Removed from types API in d29692b14dd.

Technically it's still accessible, but relying on TypeScript to avoid surfacing this and flagging unintended usage.